packages/react/src/components.tsx

```diff
@@ -11,5 +14,47 @@ export function createRemoteReactComponent<
   ReactComponentTypeFromRemoteComponentType<
     RemoteComponentType<Type, Props, AllowedChildren>
   > {
-  return componentType as any;
+  return memo(function ComponentWrapper(externalProps: any) {
```

Comment:
We need to create a wrapper component to parse element props and create portal. It's too late if we handle in the reconciler.